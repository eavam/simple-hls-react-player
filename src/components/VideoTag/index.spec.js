import React from 'react'
import renderer from 'react-test-renderer'
import { VideoTag } from './index.js'


test('VideoTag', () => {
  const props = {
    setVideoEl: () => {}
  }

  const component = renderer.create(
    <VideoTag {...props} />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()

})

