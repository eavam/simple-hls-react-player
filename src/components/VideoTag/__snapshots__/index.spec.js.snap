// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoTag 1`] = `
<video
  className="sc-bdVaJa fUgVbA"
  controls={false}
/>
`;
